src/views/Languages/index.jsx

```diff
@@ -99,8 +100,13 @@ const tagReposMapping = repositories =>
           ...MENTORED_BUG,
           whiteboards: `lang=${getIgnoreCase(BUGZILLA_LANGUAGES, language)}`,
         },
-        paging: {
-          ...BUGZILLA_PAGING_OPTIONS,
+        pagingGood: {
+          page: 0,
+          pageSize: BUGZILLA_PAGE_SIZE,
+        },
+        pagingMen: {
+          page: 0,
+          pageSize: BUGZILLA_PAGE_SIZE,
```

Comment:
I think the reason why paging is not working is because `BUGZILLA_PAGING_OPTIONS` was removed here. When I run the master branch and set the page size to 5, `hasNextPage` is set to true:

![Screen Shot 2019-08-19 at 10 11 39 AM](https://user-images.githubusercontent.com/3766511/63272494-25eb2480-c26a-11e9-8717-ef498ba8439a.png)

Reply:
But bugzilla paging option is just this:
`BUGZILLA_PAGING_OPTIONS = {
  page: BUGZILLA_PAGE_NUMBER,
  pageSize: BUGZILLA_PAGE_SIZE,
};`
where BUGZILLA_PAGE_SIZE is defaulted to 100 and BUGZILLA_PAGE_NUMBER is defaulted to 0. It is true that in that spot I could change it to BUGZILLA_PAGING_OPTIONS, but I don't think that will solve the issue. Did you try it on Firefox Desktop Frontend project? (I ask because you commented on the Languages part of the code, as opposed to Projects)

Reply:
> Did you try it on Firefox Desktop Frontend project? (I ask because you commented on the Languages part of the code, as opposed to Projects)

Ah, I didn't notice where I commented. I actually tried in on `localhost:5000/projects/taskcluster`. I just looked at the network tab to see the request being sent and noticed that `page` is set to `null` instead of 0. Perhaps that's the reason?

![Screen Shot 2019-08-19 at 10 56 59 AM](https://user-images.githubusercontent.com/3766511/63275810-330b1200-c270-11e9-87e4-49d55c29619f.png)

Reply:
I did some looking and that null occurs because the response before it was
 ```
pageInfo {
  "hasNextPage": false,
  "nextPage": null,
}
```
The first request is 0 and not null though and the response is the same as above.

Reply:
![image](https://user-images.githubusercontent.com/20358261/63309556-354b8b80-c2c5-11e9-842f-2327174968d0.png)
![image](https://user-images.githubusercontent.com/20358261/63309602-4dbba600-c2c5-11e9-8de6-8afd989bc919.png)

Here are comparison screenshots for the column problem. an HTML5 table appears to adjust cell width based on the content while react virtualized does not.

Reply:
Thanks for picking this up. I think I found the reason for the below.

>  Bugzilla GraphQL seems to be acting weirdly, as it is saying there are no pages left, when there are pages left, no matter the page size.

### TLDR: 

`this.load` function only fetch product with components. When you inspect the page and set the paging to be low, it seems that on the first query (product without component) you get the next page (1). However, you call `this.load` with the next page (1), the other variables (product, component) is different.

### Long thoughts:
#### Background
For bugzilla, the data stored in yaml is like this
```
products:
 - Firefox
 - Toolkit
 - Cloud Services: ['Firefox Sync: UI']
```

This means that product can have component, e.g in above case 'Firefox Sync: UI' or skip the component name.

When we query, we first only query bug for the product with no component, e.g in above case ['Firefox', 'Toolkit'] then in the [this.load function](https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/views/Project/index.jsx#L241) we fetch bug for each product with component.

The reason for that is that product and component relationship is not one to many. There could be one bug with same component, yet belongs to different product. Thus, we agreed to not combine the query.

Thus, the query done initially (in the `@graphql`) is for fetching product with no component. And [this.load function](https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/views/Project/index.jsx#L241) is meant for fetching product with component.

#### Proof of my theory
Remove the  `- Cloud Services: ['Firefox Sync: UI']` line from `ff.yaml` and set the paging size to be 5. It should work.

#### Possible fixes
_I'm throwing some ideas_

1. Make it such that when `loadNextPage` is called, it also fetches product with no component. 
```
const productWithNoComponentList = project.products.filter(product => typeof product === 'string');
    await this.fetchBugzilla(productWithNoComponentList, undefined);
```
2. Separate the paging information for each product-component query parameter. I think this is important, not only for Bugzilla, but also for Github (where we use different query for different label). Some query last longer than other, imagine how if the paging cursor have been updated to 4 by some query result of param A, yet other query of param B still haven't fetch the bug for paging 3.

Reply:
I'll get on doing fix 2 as it seems to be the better option. Thank you very much!

Reply:
actually I suppose I have to do both fixes if I'm reading this right.